package.json

```diff
@@ -18,7 +18,8 @@
   },
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "dependencies": {
-    "classnames": "^2.2.5"
+    "classnames": "^2.2.5",
+    "react-bootstrap": "^0.30.8"
   },
```

Comment:
so my understanding is making it a hard `dependency` will cause errors if there are conflicts in version downstream, but marking it is a `peerDependency` or `optionalDependency` will just warn. Since this is not dependency needed by every component (like `patternfly` is), can't we mark it as peer or optional? This seems more flexible, but maybe I am wrong.